_drafts/SC1-3-1-tables-headers-id-correct.md

```diff
@@ -0,0 +1,131 @@
+---
+rule_id: SC1-3-1-tables-headers-id-correct
+name: Tables headers-id associations correct
+test_mode: semi-automatic
+environment:  DOM Structure
+
+success_criterion:
+- 1.3.1 # Info and relationships (level A)
+
+authors:
+- Ken Petri
+---
+
+## Description
+
+This rule checks that data table headers-id associations reference correct targets in the correct order.
```

Comment:
In latest pull, I created a dependency on tables-headers-id-valid rule. Seems like there needs to be a first pass that validates the associations in terms of pure conformance to spec (or, maybe this would be caught by the parser?? and there is no need for the tables-headers-id-value rule???)